independent-projects/tools/common/src/test/resources/templates/basic-rest/kotlin/native-test-resource-template.ftl

```diff
@@ -1,6 +1,6 @@
 package ${package_name}
 
-import io.quarkus.test.junit.SubstrateTest
+import io.quarkus.test.junit.NativeImageTestTest
```

Comment:
This and a few others below looks wrong. `Test` is repeated.

Reply:
Good catch.

Reply:
done

Reply:
wasn't fixed yet? :)

Reply:
I might have walked on my feet myself after changing machines :-(

Thanks for catching this.